app/models/blog.rb

```diff
@@ -10,7 +10,8 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    keyword = "%#{sanitize_sql(term)}%"
+    where('title LIKE ? OR content LIKE ?', keyword, keyword)
```

Comment:
```suggestion
    where('title LIKE :hoge OR content LIKE :hoge', ...)
```

みたいなパラメータを使うと少しDRYにできます。

Reply:
パラメータで重複を回避しました。
```ruby
scope :search, lambda { |term|
    keyword = "%#{sanitize_sql(term)}%"
    where('title LIKE :search_term OR content LIKE :search_term', search_term: keyword)
  }
  ```